app/models/commissioning_document_template/standard.rb

```diff
@@ -0,0 +1,28 @@
+module CommissioningDocumentTemplate
+  class Standard < CommissioningDocumentTemplate::Base
+    def request_type
+      :Standard
+    end
+
+    def deadline
+      calculate_deadline(5)
+    end
+
+    def context
+      request_info = data_request_area.data_requests.map do |request|
```

Comment:
Some of the existing templates add information like aliases and pnc number. Is this not required anymore?

Reply:
Looking at the designs for the standard template pnc is not required. However MAPPA still uses pnc

Reply:
You could make `request_info` it's own method.